simulation/traffic_simulator/include/traffic_simulator/hdmap_utils/hdmap_utils.hpp

```diff
@@ -204,7 +204,7 @@ class HdMapUtils
 
   auto getLeftLaneletIds(
     const lanelet::Id, const traffic_simulator::RoutingGraphType,
-    const bool include_opposite_direction = true) const -> lanelet::Ids;
+    const bool include_opposite_direction = false) const -> lanelet::Ids;
```

Comment:
Changing the default argument bool may cause unintended behavior changes in C++ scenarios.
It would be desirable to eliminate the default argument and make it an error if the default argument is used, so that the user can set the behavior correctly.

Reply:
Please check the implementation changes of `getLeftLaneletIds`.
Default behavior is not changed.

Reply:
I fixed in [`2ebc41a` (#1468)](https://github.com/tier4/scenario_simulator_v2/pull/1468/commits/2ebc41a90f1eeb5443410c951f8b3f1654a6d54f)